Pokepedia.Domain/Validation/LocationByName.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Pokepedia.Domain.Validation
```

Comment:
Validation/Location/LocationByName (location or maybe LocationData, play with the naming).